config.sample.yaml

```diff
@@ -31,6 +31,8 @@ ircService:
       ssl: true
       # Whether or not IRC server is using a self-signed cert or not providing CA Chain
       sslselfsign: false
+      # The server password. Optional.
```

Comment:
Specifically this is the password sent in the `PASS` command - we should explicitly state this e.g.

> The password to send for all clients as a `PASS` command.